Oh, TypeScript makes a complaint I don't understand:
```
error TS18028: Private identifiers are only available when targeting ECMAScript 2015 and higher.
```
because we define `target: 'esnext'` in our `tsconfig.json`.

--- a/bin/packages/check-build-type-declaration-files.js
+++ b/bin/packages/check-build-type-declaration-files.js
@@ -69,13 +69,16 @@ async function getDecFile( packagePath ) {
 
 async function typecheckDeclarations( file ) {
 	return new Promise( ( resolve, reject ) => {
-		exec( `npx tsc --noEmit ${ file }`, ( error, stdout, stderr ) => {
-			if ( error ) {
-				reject( { file, error, stderr, stdout } );
-			} else {
-				resolve( { file, stdout } );
+		exec(
+			`npx tsc --target esnext --moduleResolution node --noEmit ${ file }`,
+			( error, stdout, stderr ) => {
+				if ( error ) {
+					reject( { file, error, stderr, stdout } );
+				} else {
+					resolve( { file, stdout } );
+				}
 			}
-		} );
+		);
 	} );
 }
 

